check/test.go

```diff
@@ -1,194 +0,0 @@
-// Copyright © 2017 Aqua Security Software Ltd. <info@aquasec.com>
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package check
-
-import (
-	"fmt"
-	"os"
-	"regexp"
-	"strconv"
-	"strings"
-)
-
-// test:
-// flag: OPTION
-// set: (true|false)
-// compare:
-//   op: (eq|gt|gte|lt|lte|has)
-//   value: val
-
-type binOp string
-
-const (
-	and binOp = "and"
-	or        = "or"
-)
-
-type testItem struct {
-	Flag    string
-	Output  string
-	Value   string
-	Set     bool
-	Compare compare
-}
-
-type compare struct {
-	Op    string
-	Value string
-}
-
-type testOutput struct {
-	testResult   bool
-	actualResult string
-}
-
-func (t *testItem) execute(s string) *testOutput {
-	result := &testOutput{}
-	match := strings.Contains(s, t.Flag)
-
-	if t.Set {
-		var flagVal string
-		isset := match
-
-		if isset && t.Compare.Op != "" {
-			// Expects flags in the form;
-			// --flag=somevalue
-			// --flag
-			// somevalue
-			//pttn := `(` + t.Flag + `)(=)*([^\s,]*) *`
-			pttn := `(` + t.Flag + `)(=)*([^\s]*) *`
```

Comment:
Need to check why we removed this comma in the regexp, as the version in bench_common still has the version with the comma